src/openfermion/resource_estimates/pbc/thc/utils/__init__.py

```diff
@@ -0,0 +1,20 @@
+# coverage: ignore
```

Comment:
is there a better name than "utils" for all these modules. Everything should have some utility :)

Reply:
no utils!